py/nightwatch/script.py

```diff
@@ -236,27 +241,53 @@ def main_run(options=None):
         cameras = args.cameras.split(',')
     else:
         cameras = None
+        
+    with tempfile.TemporaryDirectory() as tmpdr:
```

Comment:
Under normal usage, I like the use of the context manager to automatically cleanup the tempdir when done (including if there was an error), but for development and debugging, it would be useful to have an option to either *not* use a temporary directory or leave the directory behind upon error so that we can inspect it.  I'm not sure of a clean python syntax way to do that except for creating our own context manager with extra options.  Let's discuss.